ibm_catalog.json

```diff
@@ -0,0 +1,164 @@
+                "solution"
+            ],
+            "short_description": "Deploy your high performance computing (HPC) cluster with IBM scheduling software for compute intensive workloads.",
+            "long_description": "**Before you begin deploying IBM Cloud HPC, make sure that you meet the prerequisites listed in [the step-by-step guide](https://cloud.ibm.com/docs/allowlist/hpc-service?topic=hpc-service-before-you-begin-deploying).**\n\nIBM Cloud HPC is a deployable architecture where you can deploy both cloud compute resources (vCPU resources) and HPC scheduling software for your compute-intensive HPC workloads. You can reserve capacity on a recurring hourly basis from a dedicated IBM Cloud HPC resource pool.",
+            "provider_name": "IBM",
+            "offering_docs_url": "https://cloud.ibm.com/docs/allowlist/hpc-service?topic=hpc-service-overview",
+            "features": [
+                {
+                    "title": "Reserve HPC compute on an hourly basis",
+                    "description": "Reserve HPC compute capacity on a recurring hourly basis. This reserved time slot and capacity cannot be preempted by IBM."
+                },
+                {
+                    "title": "HPC infrastructure bundled with LSF scheduling software",
+                    "description": "Create a cluster with IBM Spectrum LSF in its own subnet and security group for HPC administrators to log in, submit, and manage HPC jobs."
+                },
+                {
+                    "title": "Automatic compute scaling with LSF resource connector",
+                    "description": "Automatically scale up compute resources based on workload demand. Compute resources are automatically scaled down when no longer required."
+                }
+            ],
+            "flavors": [
+                {
+                    "label": "Advanced",
+                    "name": "advanced",
+                    "install_type": "fullstack",
+                    "working_directory": "solution/hpc",
+                    "compliance": {
+                        "authority": "scc-v3",
+                        "profiles": [
+                            {
+                                "profile_name": "IBM Cloud Framework for Financial Services",
+                                "profile_version": "1.5.0"
+                            }
+                        ]
+                    },
+                    "release_notes_url": "https://cloud.ibm.com/docs/allowlist/hpc-service?topic=hpc-service-release-notes",
+                    "configuration": [
+                        {
+                            "key": "ibmcloud_api_key",
+                            "type": "password",
+                            "description": "IBM Cloud API Key that will be used for authentication in scripts run in this module. Only required if certain options are required.",
+                            "required": true
+                        },
```

Comment:
I see no reason why you have `ibmcloud_api_key` here - its already in your variables.tf file so not needed here unless you wanted to add exrta customisation, which you are not.

Reply:
oh sorry, I did not realise this was not required in terraform (default value of `null`) - thats ok then